packages/core/useFileDialog/index.ts

```diff
@@ -0,0 +1,53 @@
+import { readonly, ref } from 'vue-demi'
+
+export interface UseFileDialogOptions {
+  multiple?: boolean
+  accept?: string
+}
+
+const DEFAULT_OPTIONS: UseFileDialogOptions = {
+  multiple: true,
+  accept: '*',
+}
+
+/**
+ * Open file dialog with ease.
+ *
+ * @see https://vueuse.org/useFileDialog
+ * @param options
+ */
+export function useFileDialog(options?: Partial<UseFileDialogOptions>) {
+  const files = ref<FileList | null>(null)
+
+  const input = document.createElement('input')
+  input.type = 'file'
+
+  input.onchange = (event: Event) => {
+    const result = event.target as HTMLInputElement
+    files.value = result.files
+  }
+
+  const open = (localOptions?: Partial<UseFileDialogOptions>) => {
+    const _options = {
+      ...DEFAULT_OPTIONS,
+      ...options,
+      ...localOptions,
+    }
+    input.multiple = _options.multiple!
+    input.accept = _options.accept!
+
```

Comment:
```suggestion
    input.multiple = _options.multiple!
    input.accept = _options.accept!
    input.capture = _options.capture!

```
When `capture` value is `user` or `environment` or other you can select the device to limit the input source

[HTMLInputElement Capture](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/capture)